Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/Plugins/StandardPlugins/Source/AnimGraph/BlendGraphWidget.cpp

```diff
@@ -1282,6 +1245,99 @@ namespace EMStudio
         }
     }
 
+    void BlendGraphWidget::CreateNodeGroup()
+    {
+        AZ_Assert(sender()->inherits("QAction"), "CreateNodeGroup called apart from a connection to a QAction's signal");
```

Comment:
Nit: If this method gets called normally (e.g. not triggered from a signal) then `sender()` will be `nullptr` so might want to safe-guard this.